lib/rubocop/cop/lint/ensure_return.rb

```diff
@@ -3,6 +3,9 @@
 module RuboCop
   module Cop
     module Lint
+      # Explicit return from an ensure block alters the control flow
+      # as the return will take precedence over any exception being raised,
+      # and the exception will be silently thrown away.
```

Comment:
This is very clear 👍 

I'm trying to think of a way to include the word "rescue" to help even more. Perhaps something like:

> ... silently thrown away as if it were rescued.

Reply:
That's more clear and better. I will add that :+1: